hawtio-system/src/main/java/io/hawt/jmx/JmxTreeWatcher.java

```diff
@@ -61,7 +61,7 @@ public void init() throws Exception {
             if (Strings.isNotBlank(text)) {
```

Comment:
I think you can remove this `if` statement.

Reply:
To be fair, the whole thing can now probably just be boiled down to `LOG.info("Welcome to hawtio " +  getVersion());`

Reply:
Use

    LOG.info("Welcome to hawtio {}", getVersion());

instead of

    LOG.info("Welcome to hawtio " + getVersion());